addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner.js

```diff
@@ -154,9 +157,18 @@ function scan(helper,msg) {
 }
 
 function raiseAlert(helper, evidence){
-    helper.newAlert()
+    getAlertWrapper(helper, evidence).raise();
+}
+
+function getAlertWrapper(helper, evidence){
+    if (!helper) {
+        // Will only work for creating alerts, not raising them
+        helper = new WebSocketScanHelperImpl(new WebSocketPassiveScanThread(null));
+    }
+    return helper.newAlert()
+        .setPluginId(getId())
         .setRiskConfidence(RISK_MEDIUM, CONFIDENCE_MEDIUM)
-        .setName("Application Error Disclosure via WebSockets (script)")
+        .setName("Application Error Disclosure via WebSockets")
```

Comment:
I think it's important to be clear that this is from scripts in case users go searching for passive scan rules to turn off.